contribs/github-bot/internal/config/config.go

```diff
@@ -63,8 +63,26 @@ func Config(gh *client.GitHub) ([]AutomaticCheck, []ManualCheck) {
 				c.FileChanged(gh, "^gno.land/pkg/gnoweb/"),
 			),
 			Then: r.Or(
-				r.ReviewByUser(gh, "alexiscolin"),
-				r.ReviewByUser(gh, "gfanton"),
+				// If alexiscolin or gfanton is the author of the PR, the other must review it.
```

Comment:
Isn't this the same in functionality as the previous one, but longer?

Reply:
If it's exactly the same thing, but I felt that this part needed to be made more explicit even if it made it longer, I can roll back this change without any problem.

Reply:
>a. If Alexis or Guilhem is the author of a PR related to gnoweb, the other is assigned as a reviewer.

Doesn't this imply that approval from the other one is not necessary?

Reply:
I'm not sure, my idea was that since they want to review each other's work, we could make their reviews 'mandatory.'
And if one of them is OOO or their review isn't relevant in a particular case, they could just click the bot's skip button. 

I can modify it to make it completely optional.